Analytics-CSharp/Segment/Analytics/Analytics.cs

```diff
@@ -130,32 +108,21 @@ public async Task<string> UserIdAsync()
         /// <returns><see cref="JsonObject"/> instance of Traits</returns>
         public JsonObject Traits()
         {
-            var task = TraitsAsync();
-            task.Wait();
-            return task.Result;
-        }
-
-        /// <summary>
-        /// Retrieve the traits registered by a previous <see cref="Identify(string,Segment.Serialization.JsonObject)"/> call.
-        /// </summary>
-        /// <returns><see cref="JsonObject"/> instance of Traits</returns>
-        public async Task<JsonObject> TraitsAsync()
-        {
-            var userInfo = await store.CurrentState<UserInfo>();
             return userInfo.traits;
         }
 
+
         /// <summary>
         /// Retrieve the traits registered by a previous <see cref="Identify(string,Segment.Serialization.JsonObject)"/> call.
         /// </summary>
         /// <typeparam name="T">Type that implements <see cref="ISerializable"/></typeparam>
         /// <returns>Traits</returns>
-        public async Task<T> TraitsAsync<T>() where T : ISerializable
-        {   
-            var traits = await TraitsAsync();
-            return traits != null ? JsonUtility.FromJson<T>(traits.ToString()) : default;
+        public T Traits<T>() where T : ISerializable
```

Comment:
and here

Reply:
Annoyingly, doing the => thing with Traits makes the templated version give an error.  Unless we don't care about it, I'll leave the functions for both Traits.

Reply:
Consistency is good